### ISSUE TITLE
ValueError: need more than 2 values to unpack

### ISSUE THREAD

**Issue by tspecht** [Breakthrough/PySceneDetect#41]
Hey,

trying to run scenedetect on an EC2 intsance. I got python 3.4 running and numpy + opencv + mkvmerge installed, any idea why it can't open the file?

```
Finished processing dependencies for PySceneDetect==0.3.6
(scenedetect)ubuntu@ip-172-31-12-98:/tmp/scenedetect/PySceneDetect-0.4$ scenedetect -i /dev/input.mp4 -d content -t 30 -l -o /dev/output_scene.mkv
[PySceneDetect] Detecting scenes (content mode)...
[PySceneDetect] FATAL ERROR - could not open video /dev/input.mp4.
Traceback (most recent call last):
  File "/home/ubuntu/.virtualenvs/scenedetect/bin/scenedetect", line 9, in <module>
    load_entry_point('PySceneDetect==0.3.6', 'console_scripts', 'scenedetect')()
  File "/home/ubuntu/.virtualenvs/scenedetect/lib/python3.4/site-packages/PySceneDetect-0.3.6-py3.4.egg/scenedetect/__init__.py", line 302, in main
ValueError: need more than 2 values to unpack
```

**Comment by gpp8p** [comment]
I am also getting this error:

gpp8p@woozel:/media/gpp8p/ExtraDrive1/scene/PySceneDetect-0.4$ scenedetect -i testvideo.mp4 -d content -t 30 -o output_scene.mkv
[PySceneDetect] Detecting scenes (content mode)...
[PySceneDetect] FATAL ERROR - could not open video testvideo.mp4.
Traceback (most recent call last):
  File "/usr/local/bin/scenedetect", line 11, in <module>
    load_entry_point('PySceneDetect==0.3.6', 'console_scripts', 'scenedetect')()
  File "build/bdist.linux-x86_64/egg/scenedetect/__init__.py", line 302, in main
ValueError: need more than 2 values to unpack

**Comment by xufuou** [comment]
According the [setup.py](https://github.com/Breakthrough/PySceneDetect/blob/master/setup.py#L13) pyscenedetect is only compatible with python versions below 3.3.

**Comment by Breakthrough** [comment]
Hello all;

The real issue here is the video is not able to be opened by the OpenCV library:

    [PySceneDetect] FATAL ERROR - could not open video /dev/input.mp4.

All information after this is generated due to the above error.  Double-check that you've installed the Python OpenCV libraries correctly, and failing that, try building and installing from source (thus ensuring all dependencies are compiled).

Note that although this issue has the same cause as #16 (see the issue for some workarounds).  It will not be marked as a duplicate, however, as better error handling by PySceneDetect will be implemented in these cases, thus generating more useful (or, rather, less useless) text upon error.

**Comment by Breakthrough** [comment]
Hello all;

This should be fixed in the latest release of v0.5-beta, which has significantly improved error reporting facilities.  Please upgrade to the beta whenever you can, and the release of v0.5 is scheduled to be later this month.  If you notice any issues with the new version, please feel free to bring them to my attention.

Lastly, thank you for your submission/report, and for helping to make PySceneDetect even better - I really appreciate it!  👍
